pod/perlfunc.pod

```diff
@@ -9621,6 +9621,7 @@ C<use VERSION>, possibly removing the C<strict> and C<feature> added by
 C<use VERSION>.  C<use VERSION> does not
 load the F<feature.pm> or F<strict.pm>
 files.
+Similarly, <LC<warnings>> are enabled if C<VERSION> is 5.35.0 or higher.
```

Comment:
`C<>` here is used for function names or use statements, not needed for a link to the pragma on its own.

```suggestion
Similarly, L<warnings> are enabled if C<VERSION> is 5.35.0 or higher.
```

Reply:
Also this sentence should be moved before "Any explicit use..." and the following sentences should mention warnings as well.

Reply:
Fixed.     Note that it now states that use VERSION does not load warnings.pm.  I have no idea if that is true or not.

Reply:
That is accurate.